lua/treesitter_config.lua

```diff
@@ -0,0 +1,21 @@
+-- Nvim Treesitter config
+-- Just the basic config suggested in docs
+-- Documentation: https://github.com/nvim-treesitter/nvim-treesitter
+require'nvim-treesitter.configs'.setup {
+  ensure_installed = "all",
+  highlight = {
+    enable = true,
+  },
+  incremental_selection = {
+    enable = true,
+    keymaps = {
+      init_selection = "gnn",
+      node_incremental = "grn",
+      scope_incremental = "grc",
+      node_decremental = "grm",
```

Comment:
Still need to test all of these too.

In fact I should double check all of the mappings etc added in this pull request.

Reply:
Techdebt ticket raised https://github.com/js-jslog/dotfiles/pull/7#discussion_r607311492